h/services/search_index/_queue.py

```diff
@@ -160,43 +162,39 @@ def sync(self, limit):
             if job.kwargs.get("force", False):
                 annotation_ids_to_sync.add(annotation_id)
                 job_complete.append(job)
-                counts[Queue.Result.SYNCED_FORCED] += 1
-                counts[Queue.Result.COMPLETED_FORCED] += 1
+                counts[Queue.Result.SYNCED_FORCED.format(tag=job.tag)] += 1
+                counts[Queue.Result.SYNCED_TAG_TOTAL.format(tag=job.tag)] += 1
+                counts[Queue.Result.SYNCED_TOTAL] += 1
+                counts[Queue.Result.COMPLETED_FORCED.format(tag=job.tag)] += 1
+                counts[Queue.Result.COMPLETED_TAG_TOTAL.format(tag=job.tag)] += 1
+                counts[Queue.Result.COMPLETED_TOTAL] += 1
```

Comment:
It might be nice to have a counter object to tidy some of the nitty gritty away like:

```python
counts.increment('Synced', job.tag, FORCED)
counts.increment('Completed', job.tag, FORCED)

class Counts:
    ...
    def increment(self, prefix, tag, name):
        self.stuff[f"{prefix}/{tag}/{name}"] += 1
        self.stuff[f"{prefix}/{tag}/Total"] += 1
        self.stuff[f"{prefix}/Total"] += 1
```

Not quite sure how this would play with the contstants in the tests, but perhaps the trick would be to mock this and assert counts, or to have the raw data too: `raw[prefix][tag][name]+=1`.

Honestly we've already lost the `enum-ness` now, as these are templates, not values.